### PR TITLE
Add release notes for 0.11.0

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,50 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+ * Fix - Running choco pack on Mono on Linux will not include any files in .nupkg if there is no files element in the .nuspec - see [#2258](https://github.com/chocolatey/choco/issues/2258)
+ * Fix - Nuspec files with forward slash creates nested layer of directories when packed on Windows - see [#2166](https://github.com/chocolatey/choco/issues/2166)
+ * Fix - Prevent loading of Chocolatey GUI Licensed Extension assembly - see [#2078](https://github.com/chocolatey/choco/issues/2078)
+ * Fix - Working directory always being replaced with configured cache location when using Start-ChocolateyProcessAsAdmin - see [#2051](https://github.com/chocolatey/choco/issues/2051)
+ * Fix - Temp location can change unexpectedly when using System credentials - see [#2044](https://github.com/chocolatey/choco/issues/2044)
+ * Fix - Incorrect output regarding package files when package can't be found to be downloaded - see [#1962](https://github.com/chocolatey/choco/issues/1962)
+
+### IMPROVEMENTS
+ * TBD - see [#2266](https://github.com/chocolatey/choco/issues/2266)
```
Comment (Windos) [inline CHANGELOG.md:27]: "TBD"?

Comment (Windos) [inline CHANGELOG.md:27]: "TBD"?

**Review by Windos — COMMENTED** [review]
Should #2266 have a different title?

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Review by Windos — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,50 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
 
 **NOTE**: If you have a licensed edition of Chocolatey ("chocolatey.extension"), refer to this in tandem with [Chocolatey Licensed CHANGELOG](https://github.com/chocolatey/choco/blob/master/CHANGELOG_LICENSED.md).
 
+## [0.11.0](https://github.com/chocolatey/choco/issues?q=is%3Aclosed+milestone%3A0.11.0) (unreleased)
```
Comment (JPRuskin) [inline CHANGELOG.md:6]: Do we normally update this after merge/release?

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,50 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
 
+## [0.11.0](https://github.com/chocolatey/choco/issues?q=is%3Aclosed+milestone%3A0.11.0) (unreleased)
+### BREAKING CHANGES
+ * Disable importing / loading of DLL in the extension folder - see [#2333](https://github.com/chocolatey/choco/issues/2333)
+ * choco list -e -a returns pre-releases even when --pre is not passed - see [#2304](https://github.com/chocolatey/choco/issues/2304)
+
+### BUG FIXES
+ * Fix - Prevent creation of empty nuget.config in user appdata folder - see [#2233](https://github.com/chocolatey/choco/issues/2233)
+ * Fix - Upgrade all with remembered arguments changes packages to pre-release - see [#2111](https://github.com/chocolatey/choco/issues/2111)
+ * Fix - When reporting installed applications not all currently installed packages are correctly removed from list - see [#2092](https://github.com/chocolatey/choco/issues/2092)
```
Comment (pauby) [inline CHANGELOG.md:14]:
```suggestion
 * Fix - Software applications are being reported as not being managed by Chocolatey when their associated package is installed - see [#2092](https://github.com/chocolatey/choco/issues/2092)

```

**Comment by pauby** [comment]
One change. Otherwise LTGM!

**Pull request merged**